psi4/driver/procrouting/dft/libxc_functionals.py

```diff
@@ -129,7 +129,7 @@
 funcs.append({"name": "mPWB1K"         , "xc_functionals": {"HYB_MGGA_XC_MPWB1K"        : {}}})
 funcs.append({"name": "X1B95"          , "xc_functionals": {"HYB_MGGA_XC_X1B95"         : {}}})
 funcs.append({"name": "XB1K"           , "xc_functionals": {"HYB_MGGA_XC_XB1K"          : {}}})
-funcs.append({"name": "PW6B95"         , "xc_functionals": {"HYB_MGGA_XC_PW6B95"        : {}}})
+#funcs.append({"name": "PW6B95"         , "xc_functionals": {"HYB_MGGA_XC_PW6B95"        : {}}})  # duplicate of dict-based in hyb_functionals.py
```

Comment:
Seems to me all the hybrids should be moved to the other file?

Reply:
I'm not following. libxc_functionals are the ones where libxc provides a _XC_ and don't need intervention beyond dispersion and alias. We could use this PW6B95, but the other provides a nice complicated example.

Reply:
Just that I'm not sure why the functionals are scattered in several files.

Reply:
The other files are split based on the components. I guess we could in theory split the complete `XC` functionals exposed by libxc into the other categories, but this way they are all kept together and can be updated easier. So I guess if you want a hybrid, you have to check two files (as opposed to several), which is not that terrible.